sql/moz-fx-data-shared-prod/accounts_backend_derived/monitoring_db_counts_v1/query.sql

```diff
@@ -0,0 +1,203 @@
+    'password_change_tokens' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_password_change_tokens_v1`
+  UNION ALL
+  SELECT
+    'password_forgot_tokens' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_password_forgot_tokens_v1`
+  UNION ALL
+  SELECT
+    'paypal_customers' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_paypal_customers_v1`
+  UNION ALL
+  SELECT
+    'recovery_codes' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_recovery_codes_v1`
+  UNION ALL
+  SELECT
+    'security_events' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_security_events_v1`
+  UNION ALL
+  SELECT
+    'sent_emails' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_sent_emails_v1`
+  UNION ALL
+  SELECT
+    'session_tokens' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_session_tokens_v1`
+  UNION ALL
+  SELECT
+    'signin_codes' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_signin_codes_v1`
+  UNION ALL
+  SELECT
+    'totp' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_totp_v1`
+  UNION ALL
+  SELECT
+    'unblock_codes' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_unblock_codes_v1`
+  UNION ALL
+  SELECT
+    'unverified_tokens' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_unverified_tokens_v1`
+  UNION ALL
+  SELECT
+    'verification_reminders' AS table_name,
+    COUNT(*) AS total_rows
+  FROM
+    `moz-fx-data-shared-prod.accounts_db_external.fxa_verification_reminders_v1`
+  UNION ALL
+    (
+      SELECT
+        "accounts_with_secondary_emails" AS table_name,
+        COUNT(DISTINCT accounts.uid) AS total_rows
+      FROM
+        `moz-fx-data-shared-prod.accounts_db_external.fxa_accounts_v1` accounts
+      JOIN
+        `moz-fx-data-shared-prod.accounts_db_external.fxa_emails_v1` emails
+        ON accounts.uid = emails.uid
+      WHERE
+        emails.isPrimary = FALSE
+    )
+  UNION ALL
+    (
+      SELECT
+        "accounts_with_unverified_emails" AS table_name,
+        COUNT(DISTINCT accounts.uid) AS total_rows
+      FROM
+        `moz-fx-data-shared-prod.accounts_db_external.fxa_accounts_v1` accounts
+      JOIN
+        `moz-fx-data-shared-prod.accounts_db_external.fxa_emails_v1` emails
+        ON accounts.uid = emails.uid
+      WHERE
+        emails.isVerified = FALSE
+    )
+  UNION ALL
+    (
+      SELECT
+        "accounts_linked_to_google" AS table_name,
+        COUNT(uid) AS total_rows
```

Comment:
Are there cases where `uid` is null in this table?

Reply:
I've run it by hand a few times but not "awhile".  A NULL there would be a bug in the system.  There shouldn't be any.

Reply:
Usually the only reason to use `COUNT(expression)` rather than `COUNT(*)` is if the expression might be null and shouldn't be counted in that case.

Or if it was intended to count distinct `uid` values then it should be `COUNT(DISTINCT uid)`.